IPython/core/pylabtools.py

```diff
@@ -317,9 +319,15 @@ def find_gui_and_backend(gui=None, gui_select=None):
 
     import matplotlib
 
+    has_unified_qt_backend = matplotlib.__version__[:3] >= "3.5"
```

Comment:
```suggestion
    has_unified_qt_backend = getattr(matplotlib, '__version_info__', (0,0)) >= (3, 5)
```

I think `__version_info__` was added in 3.5 so if it's there... But I prefer to not add a string comparison in case you ever get to matplotlib 3.60.+